### ISSUE TITLE
Does not detect leaving community servers/manual disconnects

### ISSUE THREAD

**Issue by xqzba** [Kataiser/tf2-rich-presence#105]
After leaving a community server, RPC shows that I am still connected. Does not update to Main Menu.

**Comment by Kataiser** [comment]
Yeah I've had that happen a few times, the game likes to generate brand-new disconnect messages sometimes. May I see your console.log again? Also, are you using the source version or the release?

**Comment by xqzba** [comment]
> Yeah I've had that happen a few times, the game likes to generate brand-new disconnect messages sometimes. May I see your console.log again? Also, are you using the source version or the release?

[TF2RP_DESKTOP-TV3HDPL_Admin_v1.13.2_18425.log](https://github.com/Kataiser/tf2-rich-presence/files/4769426/TF2RP_DESKTOP-TV3HDPL_Admin_v1.13.2_18425.log)
[console.log](https://github.com/Kataiser/tf2-rich-presence/files/4769427/console.log)
I am using the latest source version. Thanks for the work you put into this!

**Comment by Kataiser** [comment]
Hopefully fixed with https://github.com/Kataiser/tf2-rich-presence/commit/600f5a6bcf209e20ee8e3917104b54a513df0785. Not sure why your game didn't output "Dropped xqzba from server (Server shutting down)"

> Thanks for the work you put into this!

:)

**Comment by xqzba** [comment]
> Hopefully fixed with [600f5a6](https://github.com/Kataiser/tf2-rich-presence/commit/600f5a6bcf209e20ee8e3917104b54a513df0785). Not sure why your game didn't output "Dropped xqzba from server (Server shutting down)"
> 
> > Thanks for the work you put into this!
> 
> :)

I think that message is only for self hosting servers. I was in an actual community server.

**Comment by Kataiser** [comment]
Oh right

**Comment by xqzba** [comment]
Doesn't seem to be working unfortunately
